docs/mine/lotus/split-markets-miners.md

```diff
@@ -0,0 +1,134 @@
+
+```
+[Subsystems]
+  EnableMining = false
+  EnableSealing = false
+  EnableSectorStorage = false
+  EnableStorageMarket = true
+```
+
+2. The `mining/sealing/proving` node - a `lotus-miner` process responsible to Filecoin mining, and sector storage, sealing and proving;
+
+```
+[Subsystems]
+  EnableMining = true
+  EnableSealing = true
+  EnableSectorStorage = true
+  EnableStorageMarket = false
+```
+
+When a `lotus-miner` node is started with configuration for a `markets` node, it includes a libp2p node and should be publicly exposed on the Internet and reachable/dialable for other clients to communicate with it.
+
+When a `lotus-miner` node is started with configuration for a `mining/sealing/proving` node, it does not include a libp2p node, and it doesn't have to be publicly exposed on the Internet. The `markets` node communicates with it via its JSON RPC interface.
+
+This document explains how to split your existing monolith `lotus-miner` node into multiple processes and take advantage of this new architecture, which brings more security to your mining operation as well as less potential attack surface.
+
+## Pre-requisites
+
+Before splitting the markets service process from the monolith miner process, you should backup your miner's metadata repository. You need to start the `lotus-miner` with the `LOTUS_BACKUP_BASE_PATH` env variable in order to do that.
+
+```sh
+export LOTUS_BACKUP_BASE_PATH=~/lotus-backup-location
+lotus daemon
+```
+
+```sh
+export LOTUS_BACKUP_BASE_PATH=~/lotus-backup-location
+lotus-miner run
+```
+
+## Creating a backup
+
+```sh
+export LOTUS_BACKUP_BASE_PATH=~/lotus-backup-location
+lotus-miner backup ~/lotus-backup-location/backupfile
+```
+
+## Copy config.toml and storage.json for the markets service
+
+You need to generate a `config.toml` and `storage.json` for the markets node, and have them ready for the next step.
+
+If you intend to run the `mining/sealing/proving` node on the same machine as the `markets` node, make sure that their listeners do not clash:
+```
+[API]
+  ListenAddress = "/ip4/127.0.0.1/tcp/8787/http"
+  RemoteListenAddress = "127.0.0.1:8787"
+```
+
+By default the `lotus-miner` node listens to port 2345, so in the example configuration above, we change the configuration for the `markets` node to listen to `127.0.0.1:8787`.
+
+`storage.json` should be left without storage paths, as the `markets` node won't handle anything storage related.
+
+```
+{
+  "StoragePaths": [
+  ]
+}
+```
+
+In the example commands below, we have placed the `config.toml` and `storage.json` in the `~/.lotusmarket` directory.
+
+## Initialising a markets service repository
+
+1. Create authentication tokens for the `markets` node
+
+```sh
+export APISEALER=`./lotus-miner auth api-info --perm=admin`
+export APISECTORINDEX=`./lotus-miner auth api-info --perm=admin`
+```
+
+2. Initialise the `market` node
+
+```sh
+export LOTUS_MINER_PATH=~/markets-repo-location
```

Comment:
Need to make add a warning that the `lotus-miner` command will interact with one or other repo depending on the path!